# Object in 3D world coordinates
string color
geometry_msgs/Point position
float32 area
